tritium-caffeine/build.gradle

```diff
@@ -5,18 +5,14 @@ targetCompatibility = 1.8
 
 dependencies {
 
-    api project(':tritium-api')
-    api project(':tritium-core')
```

Comment:
nice

Reply:
these are still transitively exposed as `api` via `tritium-metrics`, so consumers' view is same, but simplifies things here